docker-compose.yml

```diff
@@ -13,3 +13,9 @@ services:
     environment:
       ENV_FOR_DYNACONF: development
       PYTHONUNBUFFERED: 1
+  j40_web:
```

Comment:
Not set on these names, if there are others that are more appropriate we can change. Thx

`j40_web`
`j40_web_container`

Reply:
I personally think the names are good.